src/pages/Script/index.tsx

```diff
@@ -193,9 +197,9 @@ export const ScriptPage = () => {
                   {t('scripts.verified')}
                 </Tooltip>
               ) : null}
-              {deprecated === true ? (
-                <Tooltip trigger={<DeprecatedIcon />} placement="top">
-                  {t('scripts.deprecated')}
+              {isZeroLock === true ? (
+                <Tooltip trigger={<OwnerLessIcon />} placement="top">
+                  {t('scripts.link.ownerless_cell')}
```

Comment:
In some cases we should mark a script as deprecated, e.g. PW_Lock, the one not adopted in apps anymore, or obsolete versions of Omnilock, omnilock has been deployed multiple times and only the last one is recommended. Therefore I think `deprecated` should be retained.

Reply:
So this script although is live, but it's deprecated.
https://ckb-explorer-frontend-in-magickbase-repo-git-cb8e8d-magickbase.vercel.app/script/0x58c5f491aba6d61678b7cf7edf4910b1f5e00ec0cde2f42e0abb4fd9aff25a63/type

Reply:
Yes, `PW Lock` is not maintained anymore and it has an advance substitute -- omnilock. And no more new apps will support `PW Lock`